InternetShop/BLL/Services/UserService.cs

```diff
@@ -3,52 +3,20 @@
 using AutoMapper;
 using DAL.Entities;
 using BLL.Models;
+using DAL.Repositories;
 
 namespace BLL.Services
 {
-    public class UserService : IUserService
+    public class UserService : GenericService<UserModel, UserEntity>, IUserService
     {
-        protected readonly IUserRepository _userRepository;
+        protected readonly IGenericRepository<UserEntity> _genericRepository;
```

Comment:
What about naming? I think it should be call as userRepo. And why do you use generic repo here? But in the constructor you use UserRepository?